src/main/java/com/autotune/common/data/result/RecommendationNotification.java

```diff
@@ -0,0 +1,27 @@
+package com.autotune.common.data.result;
```

Comment:
LICENSE header missing. Added in #631 

#631 needs to be merged before #627 re-review